lib/ServerChannel.js

```diff
@@ -985,18 +986,25 @@ ServerChannel.prototype.connectPhantom = function(options) {
     phantomjs.stdout.setEncoding('utf8');
     logger = Logger.get('clients', {name: 'phantomjs'});
 
-    phantomjs.stdout.on('data', function(text) {
-        logger.log(text);
+    phantomjs.stdout.on('data', function(data) {
+        logger.log('stdout: ' + data);
```

Comment:
the argument to this function is not a string, if you don't convert to a string first you get something like this:
<Buffer 50 68 61 6e 74 6f ...